client/src/components/auth/authComponents/RegistrationForm.jsx

```diff
@@ -62,9 +56,11 @@ const RegistrationForm = () => {
     password: yup
       .string()
       .required('Please enter your password')
+      .min(8, 'Password must contain at least 8 characters')
+      .max(20, 'Password is too long')
```

Comment:
No specific reason, just thought that it might not be a good idea to not capping any limit for the password. And after I tried, password of 30 chars is kinda too long for the UI as well as for myself to remember, so I chose 20. But I can increase if we decided to.